src/modules/blocks/ticket/container-content/attendee-collection/style.pcss

```diff
@@ -0,0 +1,74 @@
+.tribe-editor__ticket__attendee-collection {
```

Comment:
It's not as important as this is the admin, but can we try and use BEM classes in the JS templates and the style?

Reply:
I copied from the Advanced / SKU part, so this came along with it.